src/main/java/Delete.java

```diff
@@ -0,0 +1,24 @@
+public class Delete extends Command {
+    int num;
+
+    Delete(int num) {
+        super();
+        this.num = num;
+    }
+
+    /**
+     * It executes the delete command by removing the specified task from the list.
+     *
+     * @param tasks Object of type TaskList.
+     * @param storage Object of type Storage.
+     * @return Returns affirmation that delete has been done.
+     */
+    public String execute(TaskList tasks, Storage storage) {
+        assert num <= tasks.getList().size() && num >= 0 : "This task number does not exist in the list. "

```

Comment:
Since `num` is the index of your task (counted as natural numbers starting from 1), should this assertion be `num` > 0 instead?